tools/rgrnastar/macros.xml

```diff
@@ -280,4 +280,90 @@
             <action name="column_names" type="metadata" default="GeneID,Counts_unstrand,Counts_firstStrand,Counts_secondStrand" />
         </actions>
     </xml>
+    <xml name="outWig">
+        <section name="coverage" title="Coverage" expanded="false">
```

Comment:
Sections should contain >1 parameters. Here its only one (conditional). I think we can remove the section ..

Reply:
You are right. I thought it was a good way to hide it but I can remove it.